src/zenml/client.py

```diff
@@ -3279,6 +3234,7 @@ def list_artifacts(
             created: Use to filter by time of creation
             updated: Use the last updated date for filtering
             name: The name of the run to filter by.
+            version: The version of the artifact to filter by.
```

Comment:
```suggestion
            version: The version of the artifact by which to filter.
```

Reply:
Technically better but we use that wording in 130 other places and I'll not bother adjusting all of them 😅